packages/@react-aria/interactions/test/useHover.test.js

```diff
@@ -369,4 +374,86 @@ describe('useHover', function () {
       expect(el.textContent).toBe('test');
     });
   });
+
+  describe('portal event bubbling', () => {
+    function PortalExample(props) {
+      let {elementType: ElementType = 'div', ...otherProps} = props;
+      let {hoverProps} = useHover(otherProps);
+      return (
+        <Provider theme={theme}>
+          <ElementType {...hoverProps} tabIndex="0">
+            <DialogTrigger>
+              <ActionButton>open</ActionButton>
+              <Dialog>
+                test
+              </Dialog>
+            </DialogTrigger>
+          </ElementType>
+        </Provider>
+      );
+    }
+
+    beforeAll(() => {
+      jest.useFakeTimers();
+    });
+    afterAll(() => {
+      jest.useRealTimers();
+    });
+
+    let matchMedia;
+    beforeEach(() => {
+      matchMedia = new MatchMediaMock();
+      // this needs to be a setTimeout so that the dialog can be removed from the dom before the callback is invoked
+      jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => setTimeout(() => cb(), 0));
+    });
+
+    afterEach(() => {
+      // Ensure we close any dialogs before unmounting to avoid warning.
+      let dialog = document.querySelector('[role="dialog"]');
+      if (dialog) {
+        act(() => {
+          fireEvent.keyDown(dialog, {key: 'Escape'});
+          fireEvent.keyUp(dialog, {key: 'Escape'});
+          jest.runAllTimers();
+        });
+      }
+
+      matchMedia.clear();
+      window.requestAnimationFrame.mockRestore();
+    });
+
+    describe.each`
+      type                | prepare               | actions
+      ${'Mouse Events'}   | ${installMouseEvent}  | ${[
+        (el) => fireEvent.mouseEnter(el, {button: 0}),
+        (el) => fireEvent.mouseLeave(el, {button: 0})
+      ]}
+      ${'Pointer Events'} | ${installPointerEvent}| ${[
+        (el) => fireEvent(el, pointerEvent('pointerover', {pointerType: 'touch'})),
```

Comment:
why touch? for the most part, i thought touch couldn't start hover, that's why there's not call to triggerHover* in any touch handlers in useHover